app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,44 @@
+footer {
+  background-color: rgb(232, 230, 230);
+  width: 95%;
+  margin: 0 auto;
+  height: 200px;
+  border-radius: 15px;
+
+    a {
+      color: #8f8f8f;
+
+      &:hover {
+        color: #55679b;
+      }
+    }
```

Comment:
For design cohesion can you use $main-grey as the color and $gold as the hover color please ?